application/forms/PreferenceForm.php

```diff
@@ -202,6 +203,25 @@ public function createElements(array $formData)
             }
         }
 
+        if (! isset($formData['theme'])
+            || $formData['theme'] === StyleSheet::DEFAULT_THEME
+            || strpos(file_get_contents(StyleSheet::getThemeFile($formData['theme'])),'prefers-color-scheme') !== false
```

Comment:
Only checking for `prefers-color-scheme` isn't enough. You'd need to check also for the extensions using `min-height`.

See, a theme not using the `min-height` extensions will always be in mode `auto`. If the user chooses `dark` or `light` then, it has no effect. I'd rather make only the `auto` option selectable in this case.